src/pybind/mgr/dashboard/frontend/cypress/e2e/common/global.feature.po.ts

```diff
@@ -49,6 +67,30 @@ And('I go to the {string} tab', (names: string) => {
   }
 });
 
+And('I go to the {string} carbon tab', (name: string) => {
+  cy.get('cds-tabs').within(() => {
+    cy.contains('button', name).click();
+  });
+});
+
 And('I wait for {string} seconds', (seconds: number) => {
   cy.wait(seconds * 1000);
 });
+
+Given('a volume is available', () => {
+  cy.request({
+    method: 'POST',
+    url: '/api/cephfs',
+    body: { name: 'testFs', service_spec: { placement: {} } },
+    headers: { Accept: 'application/vnd.ceph.api.v1.0+json' }
+  });
+});
+
+Given('a subvolume group is available', () => {
+  cy.request({
+    method: 'POST',
+    url: '/api/cephfs/subvolume/group',
+    body: { vol_name: 'testFs', group_name: 'testSubvGrp' },
+    headers: { Accept: 'application/vnd.ceph.api.v1.0+json' }
+  });
+});
```

Comment:
I think this should be in it's own helper file in the e2e/filesystem folder but when I tried to add a file these were not being recognized